src/crewai/crew.py

```diff
@@ -208,14 +209,17 @@ def _run_hierarchical_process(self) -> str:
             verbose=True,
         )
 
-        task_output = ""
+        task_output: str = ""
         for task in self.tasks:
             self._logger.log("debug", f"Working Agent: {manager.role}")
             self._logger.log("info", f"Starting Task: {task.description}")
 
-            task_output = task.execute(
+            output = task.execute(
                 agent=manager, context=task_output, tools=manager.tools
             )
+            if not task.async_execution:
+                assert output is not None
```

Comment:
Output could be none I guess?

Reply:
For non-async? I don't think so. Worst case scenario, it's gonna be an empty string.